### DISCUSSION TITLE
Trying to install on a Windows PC - get error "MSVCP140_1.dll" not found

### DISCUSSION THREAD

**Discussion by TheMarkofCain** [keepassxreboot/keepassxc#9611] [Q&A] [answered]
Hi,

I use KeePassXC on my Linux machine and I absolutely LOVE IT!!! It has literally made my life so much better!

I tried to install KeePassXC on my Windows machine but it won't let me, I get the error "MSVCP140.dll". It is a new laptop and I haven't messed around with it by installing or deleting anything. I am also not a particularly technical kind of person so I don't know what to do... Is this a problem with the program/ website/ installation package or something that I need to fix? I would have thought that the installation would have had all the .dll files that it needed.

I am using Windows 11 and I just clicked on the first button to download in the middle at the top of the screen, I also tried the Microsoft Store installation further down the screen as well as the Installer (64-Bit, Windows 10/11) and had the same issue, 

I would appreciate it if someone could please point me in the right direction and tell me what to do. Is this an issue that someone needs to fix on the website end or something that I need to do?

I did a search for "dll" before I posted this and didn't find anything as well.

Thanks everyone!

**Comment by droidmonkey** [comment, marked as answer]
Install the msvc redistribution libraries. They are linked directly on our website.

**Reply by TheMarkofCain** [reply]
Thank you for your help. I will do that.

**Reply by TheMarkofCain** [reply]
Of course, there is a link to the .exe right under the download button... I feel kind of stupid. Thanks again.
